pkg/mapper/dynamicfile/dynamicfile.go

```diff
@@ -21,9 +25,25 @@ type DynamicFileMapStore struct {
 	filename                  string
 	userIDStrict              bool
 	usernamePrefixReserveList []string
+
+	skip bool
+}
+
+// Meta is the collection of fields which should be included in all top-level
+// objects which are propagated for dynamic file mode
+type Meta struct {
```

Comment:
is this struct a duplicate? I notice another in `pkg/server/types.go` .

Reply:
removed the duplicate